openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.upgrade.clicked.json

```diff
@@ -0,0 +1,28 @@
+{
+    "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "id":" urn:uuid:13bc41f6-8406-4006-b929-c9f1ac156ada ",
+    "type":" NavigationEvent ",
```

Comment:
This event generates when user clicked on "Upgrade Verified" Button from courses menu, and this button redirects user to Payment Page. So that's how it is a NavigationEvent.

Reply:
Alright.